src/tests/unit/mock.test.ts

```diff
@@ -0,0 +1,34 @@
+import { describe, expect, expectTypeOf, it } from 'vitest';
+import { MockProvider } from '../MockProvider';
+import type { MockInterface } from '../MockProvider';
+import type { APIResponse } from '@wharfkit/session';
+
+describe('Test Mock Creation', () => {
+    it('mock data send has url', async () => {
```

Comment:
nit: I think the test names would be better if they read like English starting with "it"

Example:

```it('contains url when mock-data-send is loaded')```